### DISCUSSION TITLE
Inaccurate prediction on new data

### DISCUSSION THREAD

**Discussion by abodmadi** [talmolab/sleap#1275] [Help!] [answered]
I am using a 10 hour long dataset. I have created a top-down model, but, when try to test it with new data which the model was not trained on, the accuracy of tracking is not good enough. I want to improve the accuracy of my trained model  on new/unseen examples.

**Comment by roomrys** [comment, marked as answer]
Hi @abodmadi,

If the model is not working on another video, then the video is not similar enough from the training examples to yield good predictions. SLEAP allows users to get relatively good predictions with fewer examples with the restriction that videos should have been taken in a laboratory setting where all videos are nearly identical. If you are unable to get good predictions on another video, or on frames within the same video, then your model needs more examples to help diversify the training set.

Thanks,
Liezl

Related: #1105
